list-entry/test.js

```diff
@@ -0,0 +1,15 @@
+const firstListItem = document.querySelector("ul > li:first-child");
+const lastLiItem = document.querySelector("ul > li:last-child");
+const list = document.querySelector(".list");
+
+const fragment = new DocumentFragment(); 
+
+const ItemCat = document.createElement("li");
```

Comment:
```suggestion
const itemCat = document.createElement("li");
```
In js it is common to capitalize only classes - new Date() as an example